File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P05LAB_While_Loop/P03_SumNumbers.java
package P05LAB_While_Loop;

import java.util.Scanner;

public class P03_SumNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int number = Integer.parseInt(scanner.nextLine());
        int sum = 0;
        while (sum < number) {
            sum += Integer.parseInt(scanner.nextLine());
        }
        System.out.println(sum);
    }
}

File: JavaScript_Basic_04_2023_SolutionsOfProblems/05.LAB_PB-JS-While-Loop/P03.SumNumbers.js
function sumNumbers(input) {
    let targetSum = Number(input[0]);
    let sum = 0, i = 1;
    while (sum < targetSum) {
        sum += Number(input[i]);
        i++;
    }
    console.log(sum);
}
sumNumbers(["20",
    "1",
    "2",
    "3",
    "4",
    "5",
    "6"])
